src/index.tsx

```diff
@@ -1,6 +1,13 @@
+import React from 'react';
 import { createRoot } from 'react-dom/client';
 
+import App from './App';
+
 const container = document.getElementById('app');
 
 const root = createRoot(container!);
-root.render(<h1>Hello, World!!!</h1>);
+root.render((
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>
+));
```

Comment:
- See also: https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis